### PR TITLE
fixed indicator legend in Stock Status (map) report

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue @esoergel 

http://manage.dimagi.com/default.asp?168583#987369

### CODE REVIEW

File: corehq/apps/reports/static/reports/javascripts/maps_utils.js [hunk 1]
```diff
@@ -403,6 +403,9 @@ function makeDisplayContext(metric, setActiveFeature) {
             // store visibility on the feature object so datatables can access it
             feature.visible = (feature._conf != null);
             if (!feature.visible) {
+                if(feature.$tr === void(0)){
+                    feature.visible
```
Comment (dimagimon) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: Expected an assignment or function call and instead saw an expression.

Comment (dimagimon) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: Missing semicolon.

File: corehq/apps/reports/static/reports/javascripts/maps_utils.js [hunk 2]
```diff
@@ -403,6 +403,9 @@ function makeDisplayContext(metric, setActiveFeature) {
             // store visibility on the feature object so datatables can access it
             feature.visible = (feature._conf != null);
             if (!feature.visible) {
+                if(feature.$tr === void(0)){
```
Comment (czue) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: out of curiosity any reason you're using `void(0)` instead of `undefined`?

Comment (czue) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: also could this just be changed to `if(feature.$tr)`? it seems like if it was `null` the next line would also fail.

Comment (lwyszomi) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: ```undefined``` we can override in JS, and someone told me that I should use ```void(0)```

**Comment by lwyszomi** [comment]
@czue updated

File: corehq/apps/reports/static/reports/javascripts/maps_utils.js [hunk 3]
```diff
@@ -403,6 +403,9 @@ function makeDisplayContext(metric, setActiveFeature) {
             // store visibility on the feature object so datatables can access it
             feature.visible = (feature._conf != null);
             if (!feature.visible) {
+                if(feature.$tr){
```
Comment (czue) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: should this not be `if (!feature.$tr) {`?

Comment (lwyszomi) [inline corehq/apps/reports/static/reports/javascripts/maps_utils.js]: Yes, should be. Updated

**Pull request merged**